### PR TITLE
ETC.tsv 7882-7942

### PR DESCRIPTION
**Pull request by zaxxer**
trying to make some sense out of these

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -7938,8 +7938,8 @@ ETC_20150317_007938	Quest Buff
 ETC_20150317_007939	Move to Fallen Worry Forest through Gate Route and Spine Heart Dale
 ETC_20150317_007940	Move to Leaf Veil Plateau
 ETC_20150317_007941	I will move to Fedimian
-ETC_20150317_007942	Get leather from Jukopus
-ETC_20150317_007943	Get leather from Vubbe Thief
+ETC_20150317_007942	Get leather from Jukopus'
```
Comment (Sunaries) [inline ETC.tsv]: Why the apostrophe?

File: ETC.tsv [hunk 2]
```diff
@@ -7938,8 +7938,8 @@ ETC_20150317_007938	Quest Buff
...
+ETC_20150317_007943	Get leather from Vubbe Thiefs
```
Comment (Sunaries) [inline ETC.tsv]: Thiefs is grammatically incorrect. Either change it back to Vubbe Thief or Vubbe Thieves.

File: ETC.tsv (hunk 1 above)
Comment (zaxxer) [inline ETC.tsv]: ^this was meant to be plural as well as you're not collecting from one certain enemy but from a general group/type

**Comment by imcgames** [comment]
The Pull Request is closed for issues regarding copyrights as stated in our main page. 
We will be updating you with details in the near future regarding this matter so please hold your Pull Requests until things the issue is resolved. 
We apologize for the inconvenience caused.

**Pull request closed without merging**